clock/testing/fake_clock.go

```diff
@@ -229,6 +229,14 @@ func (f *FakeClock) HasWaiters() bool {
 	return len(f.waiters) > 0
 }
 
+// HasWaiters returns the number of waiters, if After or AfterFunc has been called or
+// if a Timer or a Ticker was created on f but not yet satisfied (so you can write race-free tests).
+func (f *FakeClock) Waiters() int {
```

Comment:
wrong function name in comment

But the comment needs to be more explicit if you want people to be able to use it reliably. Something like:

```
// Waiters returns the number of "waiters" on the clock. A waiter exists for:
//   - every call to After that has not yet signaled its channel.
//   - every call to AfterFunc that has not yet called its callback.
//   - every timer created with NewTimer which is currently ticking.
//   - every ticker created with NewTicker which is currently ticking.
//   - every ticker created with Tick.
```

and then maybe make HasWaiters say that it returns true if Waiters returns non-0.

Reply:
ack